packages/nimble-components/src/switch/styles.ts

```diff
@@ -108,7 +108,8 @@ export const styles = css`
                     var(--ni-private-switch-height) - var(
                             --ni-private-switch-indicator-size
                         )
-                ) / 2
+                ) /
+                2
```

Comment:
Do we prefer this formatting?

Reply:
I'm OOO and did not designate an owner to get the PR through. Since the repo is in a bad state (see PR description) and touching lots of files, I'll volunteer you to resolve the discussions and shepard through / own finding someone to delegate to.

Reply:
@atmgrifter00 I agree it looks goofy but I don't think it's worthwhile to block the dependency update on this, particularly it only affects a tiny number of files. If we find it bugs us as we do development we can either mark sections to be ignored by prettier, see if this setting is configurable, or re-open the debate about prettier more generally.